2024-02-13 18:04:33.260721+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "GENAIMODEL" not found (this database is empty); SQL statement:
insert into genaimodel (qanswer,timestamp,user_name,user_prompt,qid) values (?,?,?,?,default) [42104-224]
2024-02-13 18:06:21.048429+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:07:31.607499+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:08:41.801647+05:30 jdbc[13]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "SELECT * FROM GENAIMODEL GENAIMODEL [*]GENAIMODEL"; SQL statement:
SELECT * FROM GENAIMODEL GENAIMODEL GENAIMODEL [42000-224]
2024-02-13 18:09:28.484402+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:11:48.475825+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:12:14.285633+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:13:17.344644+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:14:53.555428+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:14:59.274724+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:15:24.487923+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:37:50.082599+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 18:41:14.554983+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:37:47.972346+05:30 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:52:57.222112+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:54:41.236828+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:54:59.196533+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:55:55.440224+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 19:56:37.094864+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 20:00:19.103363+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 20:04:05.348355+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 20:04:24.496039+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 21:33:34.702792+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 21:46:43.860450+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 21:56:14.999764+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 22:18:36.383205+05:30 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 22:39:19.517460+05:30 jdbc[13]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 22:45:14.343331+05:30 jdbc[23]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:07:51.453998+05:30 jdbc[33]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE [*]* FROM GENAIMODEL WHERE QID IN RANGE(1,13)"; expected "identifier"; SQL statement:
DELETE * FROM GENAIMODEL WHERE QID IN RANGE(1,13) [42001-224]
2024-02-13 23:08:22.185740+05:30 jdbc[33]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE [*]* FROM GENAIMODEL WHERE QID =1"; expected "identifier"; SQL statement:
DELETE * FROM GENAIMODEL WHERE QID =1 [42001-224]
2024-02-13 23:10:08.550444+05:30 jdbc[33]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "CUSTOMERS" not found; SQL statement:
DELETE FROM Customers WHERE QID IS 1 [42102-224]
2024-02-13 23:10:19.857852+05:30 jdbc[33]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Syntax error in SQL statement "DELETE FROM GENAIMODEL WHERE QID IS [*]1"; expected "NOT, OF, JSON, NULL, DISTINCT, TRUE, FALSE, UNKNOWN"; SQL statement:
DELETE FROM GENAIMODEL WHERE QID IS 1 [42001-224]
2024-02-13 23:12:18.570696+05:30 jdbc[32]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:12:41.993122+05:30 jdbc[34]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:13:01.363969+05:30 jdbc[34]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:19:11.393335+05:30 jdbc[34]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:39:37.799717+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "QANSWER CHARACTER VARYING(255)": "U&'Title: The Unbreakable Bond of Humanity\\000a\\000aIntroduction:\\000aHumanity i... (2757)"; SQL statement:
insert into genaimodel (qanswer,timestamp,topic,user_name,user_prompt,qid) values (?,?,?,?,?,default) [22001-224]
2024-02-13 23:42:43.374344+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLDataException: Value too long for column "QANSWER CHARACTER VARYING(255)": "U&'The Human Experience: A Journey of Triumph and Resilience\\000a\\000aIntroducti... (3250)"; SQL statement:
insert into genaimodel (qanswer,timestamp,topic,user_name,user_prompt,qid) values (?,?,?,?,?,default) [22001-224]
2024-02-13 23:44:30.782590+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:49:06.759684+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-13 23:49:26.868862+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:40:58.069198+05:30 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:41:31.838028+05:30 jdbc[14]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:43:36.143029+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:44:16.403969+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:47:42.913002+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:48:29.296359+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:49:51.133812+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:58:49.619960+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 00:59:40.108312+05:30 jdbc[24]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 01:05:52.359935+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
2024-02-14 01:07:46.815869+05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: The object is already closed [90007-224]
